draft-ietf-intarea-provisioning-domains.md

```diff
@@ -946,6 +954,13 @@ communicate with. In such scenarios, the host SHOULD check that the
 provided PvD ID, as well as the IP address that it resolves into, are
 part of the allowed whitelist.
 
+Network operators might want to restrict access to PvD Additional
+Information to only expose this to hosts that are connected to the local
+network, especially if the Additional Information would provide information
+about local network configuration to attackers. In such cases, operators
+SHOULD filter access to the Additional Information server, such as by
```

Comment:
I think "might want" can be replaced with SHOULD, and the last sentence modified:

> Network operators SHOULD restrict access to PvD Additional
 Information to only expose this to hosts that are connected to the local
 network, especially if the Additional Information would provide information
 about local network configuration to attackers. This may be implemented by
whitelisting access from the addresses and prefixes that the router provides.